oncall/customRotation.txt

```diff
@@ -0,0 +1,4 @@
+#Enter a custom shift info in the following format:
+#{Github username} {Cover start date} {Cover end date}
+#Please make sure the date is in the format YYYY-MM-DD
+harryli0108 1999-02-01 1999-02-08
```

Comment:
Probably missed this in your demo, but do we need to add all our schedules here? and curious why the start date is 1999?

Reply:
this one is only used when say you're having days off during you oncall, and you need someone to cover your rotation. The reason why it has an entry in 1999 is because that one is an example to show the format of each entry without accidentally assigning issues to the wrong person.

Reply:
Thanks for clarifying!